lib/src/util/io.dart

```diff
@@ -130,8 +130,12 @@ void warn(String message, {bool color}) {
 /// applications that don't print out the bound port.
 Future<T> getUnusedPort<T>(FutureOr<T> tryPort(int port)) async {
   T value;
+  var supportsIPv6 = false;
+  if (await _supportsIPv6()) {
+    supportsIPv6 = true;
+  }
```

Comment:
Discussed offline. Had concerns with storing state in this utility file. Given that it is guaranteed to be system wide I have modified it.